fix(css): remove old polyfills

Rotzbua opened the pull request:
<!--
## Before you hit that Submit button....

Please read our [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
before submitting a pull-request.

## Types of changes

What types of changes does your code introduce?
Put an `x` in the boxes that apply
-->

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

<!--
If you want to raise a pull-request with a new feature, or a refactoring
of existing code, it **may well get rejected** if it hasn't been discussed on
the [forum](https://discourse.nodered.org) or
[slack team](https://nodered.org/slack) first.

-->

## Proposed changes

* fix(css): remove old polyfills
  * Remove vendor specific polyfills for Konqueror, old Safari, Internet Explorer, Mozilla and Chrome
* fix(css): remove duplicates

## Checklist
<!-- Put an `x` in the boxes that apply -->

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `npm run test` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

GogoVega commented on packages/node_modules/@node-red/editor-client/src/sass/mixins.scss:
```diff
@@ -15,18 +15,10 @@
  **/
 
 @mixin disable-selection {
-    -webkit-user-select: none;
```
I wouldn't remove WebKit because Safari and Firefox browsers support it. Additionally Safari doesn't like `user-select` at all.

Rotzbua committed:
fix(css): remove old polyfills

Remove vendor specific polyfills for Konqueror, old Safari, Internet Explorer, Mozilla and Chrome

Rotzbua committed:
fix(css): remove duplicates

Remove IE hack

knolleary commented:
This PR is removing a lot of `-webkit` that is still required for Opera.
